index.js

```diff
@@ -61,6 +68,13 @@ exports.register = function(server, options, next) {
         if (request.query.json === '1') {
           return reply(combinedData).type('application/json');
         }
+
+        if (typeof options.preResponse === 'string') {
+          return str2fn(server.methods, options.preResponse)(request, options, combinedData, reply, () => {
+            reply.view(viewConfig.view, combinedData);
```

Comment:
should it do some sort of check if it should render the view? How do we have it redirect?